plugins/node/opentelemetry-instrumentation-mysql/examples/package.json

```diff
@@ -34,12 +34,12 @@
     "@opentelemetry/api": "^1.0.0",
     "@opentelemetry/exporter-jaeger": "^1.0.0",
     "@opentelemetry/exporter-zipkin": "^1.0.0",
-    "@opentelemetry/instrumentation": "^0.45.0",
-    "@opentelemetry/instrumentation-http": "^0.45.0",
+    "@opentelemetry/instrumentation": "^0.45.1",
+    "@opentelemetry/instrumentation-http": "^0.45.1",
     "@opentelemetry/instrumentation-mysql": "^0.31.0",
     "@opentelemetry/sdk-trace-base": "^1.0.0",
     "@opentelemetry/sdk-trace-node": "^1.0.0",
-    "@opentelemetry/exporter-metrics-otlp-grpc": "0.45.0",
+    "@opentelemetry/exporter-metrics-otlp-grpc": "0.45.1",
```

Comment:
does anyone know why this on is pinned but all the other OTel deps are not?
Likely not a big deal as it is only an example.

Reply:
There's no reason AFAIK, I thin it's better to align them, I opened a PR: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1784